src/main/java/com/redhat/pantheon/servlet/ModuleListingServlet.java

```diff
@@ -39,16 +39,27 @@
     protected void doGet(@Nonnull SlingHttpServletRequest request, @Nonnull SlingHttpServletResponse response) throws ServletException, IOException {
         ModuleDataRetriever mdr = new ModuleDataRetriever(request.getResourceResolver());
         String searchParam = getParam(request, "search");
+        String countParam = getParam(request, "search");
```

Comment:
I suspect this is a typo and that you intended to get the param of `count` instead, but I think we shouldn't need a count param at all. I see that this variable is unused, so please remove it.